lib/pbench/test/unit/server/test_datasets_metadata.py

```diff
@@ -16,19 +16,21 @@ def test_get_no_dataset(self, client, server_config, attach_dataset):
             },
         )
         assert response.status_code == HTTPStatus.BAD_REQUEST
-        assert response.json == {"message": "Dataset foobar not found"}
+        assert response.json == {"message": "Dataset node>foobar not found"}
 
     def test_get_bad_keys(self, client, server_config, attach_dataset):
         response = client.get(
             f"{server_config.rest_uri}/datasets/metadata",
             query_string={
                 "controller": "node",
-                "name": "foobar",
+                "name": "drb",
                 "metadata": ["xyzzy", "plugh", "owner", "access"],
             },
         )
         assert response.status_code == HTTPStatus.BAD_REQUEST
-        assert response.json == {"message": "Dataset foobar not found"}
+        assert response.json == {
+            "message": "Unrecognized list values ['xyzzy', 'plugh'] given for parameter metadata; expected saved,seen,user,deletion,access,owner"
```

Comment:
You added code to sort the keywords...should you do the same for lists?

Reply:
It would be consistent, so probably. I didn't add keyword sorting for any particularly altruistic reason; introducing your idea of set operations changed the order and I figured I should tie that loose end down. 😁